packages/monaco/src/browser/monaco-editor.ts

```diff
@@ -139,6 +141,28 @@ export class MonacoEditor implements TextEditor {
                 event: e.event.browserEvent
             });
         }));
+        this.toDispose.push(codeEditor.onDidScrollChange(e => {
+            this.onScrollChangedEmitter.fire(undefined);
+        }));
+    }
+
+    getVisibleLineRange(): Range | undefined {
```

Comment:
@a1994846931931, please just make use of `MonacoEditor.getControl().getVisibleRanges()` to implement this. also align naming.

Reply:
Thanks! Updated.